shopelectro/views/catalog.py

```diff
@@ -25,6 +25,13 @@ def get_products_count(request):
     return PRODUCTS_ON_PAGE_MOB if mobile_view else PRODUCTS_ON_PAGE_PC
 
 
+def get_paginated_page(objects, per_page, page_number):
+    try:
+        return Paginator(objects, per_page).page(page_number)
```

Comment:
why not to use `Paginator(...).page(...)` directly?
If we are needed to explain class's method with function wrapper, we have problems with class and/or it's method

Reply:
@duker33 , 
>why not to use Paginator(...).page(...) directly?

because we have to catch an `InvalidPage` error here.

>we have problems with class and/or it's method

It is a django predefined class, so it is not our an issue.

Reply:
@artemiy312 , so, maybe to extend this class with `page_or_404` method?
Decide